release-notes/9.0/preview/preview7/api-diff/Microsoft.NETCore.App/9.0-preview7_System.Security.Cryptography.md

```diff
@@ -0,0 +1,43 @@
+# System.Security.Cryptography
+
+``` diff
+ namespace System.Security.Cryptography {
+-    [UnsupportedOSPlatformAttribute("browser")]
+-    [UnsupportedOSPlatformAttribute("ios")]
+-    [UnsupportedOSPlatformAttribute("tvos")]
+-    public sealed class AesGcm : IDisposable
++    [SupportedOSPlatformAttribute("ios13.0")]
++    [SupportedOSPlatformAttribute("maccatalyst")]
++    [SupportedOSPlatformAttribute("tvos13.0")]
++    [UnsupportedOSPlatformAttribute("browser")]
++    [UnsupportedOSPlatformAttribute("ios")]
++    [UnsupportedOSPlatformAttribute("tvos")]
++    public sealed class AesGcm : IDisposable
```

Comment:
The diff is weird... lines 5-7 and 12-14 are "remove this, and put it back", so could be eliminated.  Line 8/15 is just "context", not "diff".  But doesn't really matter.